### ISSUE TITLE
random chance

### ISSUE THREAD

**Issue by Friskzips** [AntoninJuquel/lethal-company-lockout#4]
My friends like this mod, but according to them, it should be a random chance to lockdown, maybe a warning when you pull the lever to start that tells you that this building has a lockdown feature.

**Comment by AntoninJuquel** [comment]
Interesting idea I ll let you know when I implement this feature, It could be as follow in the config file
_RandomChanceToLockdown: 1_ (0 to 1) 
_eg._ 0.5 to be a 50% chance that each planet can have a lockdown featrure 
And it is annouced at the start of the level right ?

**Comment by Friskzips** [comment]
Thanks for the quick response.

Yes, I was thinking exactly like that, and it would be cool if during the announcement it told you the time when it locks and the time when it unlocks.

**Comment by Index154** [comment]
I also think this would be a great feature to have. Would be nice for adding some variety to a modpack without changing the way you have to play the game all of the time.

**Comment by Friskzips** [comment]
Yeah, it would be nice, but I don't think that the creator is working on this anymore.
